shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -643,8 +645,6 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
     lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_CREATE);
 
-    registerOnBackInvokedCallback();
```

Comment:
Now that this is removed, it means that by default, Flutter will not handle back gestures.  So if you start up a Flutter app that does nothing, has no Navigator or WidgetsApp etc., root predictive back will work.  The framework will have to first call setFrameworkHandlesBacks in order to prevent predictive back and to handle back gestures itself.  This will be handled automatically in Flutter apps that use WidgetsApp.